tests/EmitTest.elm

```diff
@@ -45,6 +45,12 @@ javascript =
                     , ( "negative int", Literal (Int -998), "-998" )
                     ]
                 )
+            , describe "Float"
+                (List.map runTest
+                    [ ( "positive float", Literal (Float 12.3), "12.3" )
+                    , ( "negative float", Literal (Float -12.3), "-12.3" )
```

Comment:
Can we please add a cornercase of `Float 0.0` and `Float -0.0`? Are they even different?
As we use `String.fromFloat` we're getting its behaviour, I'm not sure what it actually is :smile: